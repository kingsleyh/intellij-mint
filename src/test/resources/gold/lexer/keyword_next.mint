component A {

  fun woop : Void {
    next { cat = cat, dog = dog }
  }

  fun render : Html {
    </div>
  }

}