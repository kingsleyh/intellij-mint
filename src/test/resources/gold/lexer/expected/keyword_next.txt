component ('component')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('A')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
LOWER_IDENT ('woop')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('Void')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
next ('next')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE (' ')
LOWER_IDENT ('cat')
WHITE_SPACE (' ')
= ('=')
WHITE_SPACE (' ')
LOWER_IDENT ('cat')
, (',')
WHITE_SPACE (' ')
LOWER_IDENT ('dog')
WHITE_SPACE (' ')
= ('=')
WHITE_SPACE (' ')
LOWER_IDENT ('dog')
WHITE_SPACE (' ')
} ('}')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
LOWER_IDENT ('render')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('Html')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
< ('<')
/ ('/')
LOWER_IDENT ('div')
> ('>')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n')
} ('}')